drivers/pds/targetcluster/targetcluster.go

```diff
@@ -47,35 +48,60 @@ type TargetCluster struct {
 	kubeconfig string
 }
 
-// RegisterToControlPlane register the target cluster to control plane.
-func (targetCluster *TargetCluster) RegisterToControlPlane(controlPlaneURL string, helmChartversion string, bearerToken string, tenantID string, clusterType string) error {
-	log.Info("Test control plane url connectivity.")
-	_, err := isReachbale(controlPlaneURL)
-	if err != nil {
-		return fmt.Errorf("unable to reach the control plane with following error - %v", err)
-	}
+//RegisterToControlPlane register the target cluster to control plane.
+func (targetCluster *TargetCluster) RegisterToControlPlane(controlPlaneURL string, helmChartversion string, bearerToken string, tenantId string, clusterType string) error {
 	var cmd string
-	apiEndpoint := fmt.Sprintf(controlPlaneURL + "/api")
-	log.Infof("Installing PDS ( helm version -  %v)", helmChartversion)
-	if strings.EqualFold(clusterType, "ocp") {
-		cmd = fmt.Sprintf("helm install --create-namespace --namespace=%s pds pds-target --repo=https://portworx.github.io/pds-charts --version=%s --set platform=ocp --set tenantId=%s "+
-			"--set bearerToken=%s --set apiEndpoint=%s --kubeconfig %s", PDSNamespace, helmChartversion, tenantID, bearerToken, apiEndpoint, targetCluster.kubeconfig)
+	apiEndpoint := fmt.Sprintf(controlPlaneURL + "api")
+	log.Infof("Verify if the namespace %s already exits.", PDSNamespace)
+	isRegistered := false
+	_, err := k8sCore.GetNamespace(PDSNamespace)
+	if err != nil {
+		log.InfoD("Namespace %v doesnt exist ", PDSNamespace)
```

Comment:
don't you need to handle the error?

Reply:
This check is not needed, I have removed it